test/katex-spec.js

```diff
@@ -2651,6 +2651,9 @@ describe("An aligned environment", function() {
 describe("operatorname support", function() {
     it("should not fail", function() {
         expect("\\operatorname{x*Π∑\\Pi\\sum\\frac a b}").toBuild();
+        expect("\\operatorname*{x*Π∑\\Pi\\sum\\frac a b}").toBuild();
+        expect("\\operatorname*{x*Π∑\\Pi\\sum\\frac a b}_y x").toBuild();
+        expect("\\operatorname*{x*Π∑\\Pi\\sum\\frac a b}\\limits_y x").toBuild();
```

Comment:
What's the difference between the last two examples?  It might be good to have a screenshot test with those two items.

Reply:
It should build with or without `\limits`. So I wrote a test case both ways. And yes, a screen shot both ways would help.

Reply:
Thanks for updating the screenshots to check for this.